docs/topics/jvm/jvm-records.md

```diff
@@ -61,14 +56,4 @@ To declare a data class with the `@JvmRecord` annotation, it must meet the follo
 * The class cannot declare any properties with backing fields – except those initialized from the corresponding primary constructor parameters.
 * The class cannot declare any mutable properties with backing fields.
 * The class cannot be local.
-* The primary constructor of the class must be as visible as the class itself.
-
-## Enabling JVM records
```

Comment:
`@JvmRecord` should be available when jvmTarget is explicitly set to 16 or automatically inferred to 16, for example, when running Gradle under that Java version.  While it is listed in requirements, it may be worth to reiterate it in this dedicated section.

Reply:
I returned the section and add links to the corresponding compiler options in Gradle and Maven.